Car.java

```diff
@@ -0,0 +1,13 @@
+public class Car {
+    public static void start(){

```

Comment:
1. Please apply formatting to this class by pressing ctrl+alt+L 
2. Let's make out program a little more interesting. Let's add println() statement to each method via message like "Electricity is started", "Fuel system is started" etc. In the method start() in the end, put another print statement via the message "Engine is started"